metadata.json

```diff
@@ -35,29 +35,26 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "10",
         "11"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "18.04",
-        "20.04"
+        "20.04",
```

Comment:
please split the changes into two PRs, one that's a breaking change and one that's an enhancement.

Reply:
okay then :(